workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -11,5 +11,15 @@ public class SpatialOSComponent : MonoBehaviour
         public Entity Entity;
         public World World;
         public WorkerSystem Worker;
+
+        public bool IsEntityOnThisWorker(EntityId entityId)
+        {
+            return Worker.EntityIdToEntity.ContainsKey(entityId);
+        }
+
+        public bool IsEntityOnThisWorker(Entity entity)
+        {
+            return Worker.EntityIdToEntity.ContainsValue(entity);
```

Comment:
This will be super heavy

Reply:
I'm not sure the utility of an `Entity` overload for this anyway

Reply:
agreed, do we need this is or is the entity id version enough for now?

Reply:
+1 on it being heavy

I think EntityId version should be enough for now, but I left in the Entity version in case some user wants to use the heavy method - but do we want to let users even do that?

Reply:
I'll remove the heavy Entity version, to discourage users from doing bad things

Reply:
Yup - also not sure of the use case where you have the `Entity` but not the EntityId (or have a quick way of accessing it)